### ISSUE TITLE
Hash character not encoded for proper link in main list

### ISSUE THREAD

**Issue by carueda** [mmisw/orr-portal#155]
From https://github.com/mmisw/orr-ont/issues/73

----

Ok, so it's the one listed as

         https://mmisw.org/ont/~mohammedajooz/AWWM#

which, when clicked, you get

>![image](https://user-images.githubusercontent.com/556505/78396638-90a46280-75a4-11ea-895c-4b1820effce2.png)

The trailing hash `#` has been "lost" because it's not being encoded (`%23`).

So, this is a regression as we have addressed similar issue in the past (a related entry is (https://github.com/mmisw/mmiorr/issues/213).

BTW, with proper encoding, I can see that there's an ontology registered by the IRI `https://mmisw.org/ont/~mohammedajooz/AWWM#`. 

This is hard to see with a browser, but on the command line:

```
$ http -pb get 'https://mmisw.org/ont/~mohammedajooz/AWWM%23'
<?xml version="1.0"?>
...
</rdf:RDF>

<!-- Generated by the OWL API (version 3.4.2) http://owlapi.sourceforge.net -->
```

**Comment by carueda** [comment]
Interesting, with running the orr-portal locally, it seems the appropriate encoding is actually in place:

> 
![image](https://user-images.githubusercontent.com/556505/78400126-f98ed900-75aa-11ea-845a-ebcd7f081ec1.png)

Including correct handling when the link clicked as expected:

>
![image](https://user-images.githubusercontent.com/556505/78400285-3eb30b00-75ab-11ea-9949-8854e565d599.png)

However, the overall URL dispatch is different when in development mode because the prefix is different from the actual URL of the ORR repository and ("fully-hosted") ontologies.

So, will see what's getting in the way there.

**Comment by carueda** [comment]
Diagnostics:   Yes, the no-hash-encoding issue only happens for "fully-hosted" ontologies

![image](https://user-images.githubusercontent.com/556505/78401213-12988980-75ad-11ea-85b9-6fb221d469c8.png)

**Comment by carueda** [comment]
orr-portal part of the fix now in place (new version 3.9.0)

> ![image](https://user-images.githubusercontent.com/556505/78406562-bbe47d00-75b7-11ea-9dbf-bfb662ad5d90.png)

But there's still some back-end handling to be fixed (when the request is from a browser) as this IRI does exist as shown with the HTTPie command above.

**Comment by carueda** [comment]
3.9.1 enabled for the MMI ORR.  (@graybeal )

Now all self-hosted ontology entries having a `#` in the IRI are properly assigned encoded links for corresponding resolution.

A handy way to test this is to enter `#` in the filter field and then inspect the relevant entries, which are those having `https://mmisw.org/ont` as prefix.  (All other non-self-hosted should continue to be dispatched properly.)

![image](https://user-images.githubusercontent.com/556505/78414730-5d78c800-75d2-11ea-8bab-3f6d1ec5722b.png)
